elixir/apps/domain/lib/domain/instrumentation.ex

```diff
@@ -1,23 +1,28 @@
 defmodule Domain.Instrumentation do
+  alias Domain.Accounts
+  alias Domain.Actors
   alias Domain.Clients
   alias Domain.GoogleCloudPlatform
 
   def create_remote_log_sink(%Clients.Client{} = client) do
     config = config!()
-    enabled? = Keyword.fetch!(config, :client_logs_enabled)
 
-    if enabled? and GoogleCloudPlatform.enabled?() do
+    with {:ok, actor} <- Actors.fetch_actor_by_id(client.actor_id),
+         {:ok, account} <- Accounts.fetch_account_by_id(actor.account_id),
+         true <- Keyword.fetch!(config, :client_logs_enabled),
+         true <- GoogleCloudPlatform.enabled?() do
       now = DateTime.utc_now() |> DateTime.to_iso8601()
 
       bucket =
         Application.fetch_env!(:domain, __MODULE__)
         |> Keyword.fetch!(:client_logs_bucket)
 
-      filename = "clients/#{client.id}/#{now}-#{System.unique_integer([:positive])}.json"
+      filename =
+        "clients/#{account.slug}/#{actor.name}/#{client.id}/#{now}-#{System.unique_integer([:positive])}.json"
```

Comment:
@jamilbk `actor.name` can be changed
```suggestion
        "clients/#{client.account_id}/#{client.actor_id}/#{client.id}/#{now}-#{System.unique_integer([:positive])}.json"
```
and we don't have to fetch `actor` from DB every X minutes.

Same goes for account slugs, it's better to use the ID.

Reply:
It's ok if it changes. This needs to be something I can quickly browse through to find relevant logs from a customer with issues. We probably should be deleteing logs after 30 days anyway.

Reply:
Yeah I was wondering if those values may be already on the socket or available in some process' memory already?

Reply:
> We probably should be deleteing logs after 30 days anyway.

We delete them in 3 days right now: https://github.com/firezone/firezone/blob/main/terraform/environments/production/main.tf#L362

> Yeah I was wondering if those values may be already on the socket or available in some process' memory already?

You can read account from `socket.assigns.subject.account`, `actor` won't be present.

> This needs to be something I can quickly browse through to find relevant logs from a customer with issues.

We need an admin panel 🤣

Reply:
@jamilbk let's do:
```elixir
account_slug = socket.assigns.subject.account.slug
actor_name = socket.assigns.subject.actor.name |> String.downcase() |> String.replace(" ", "_") |> String.replace(~r/[^a-zA-Z0-9_-]/iu, "")

"clients/#{account_slug}/#{actor_name}/#{client.id}/#{now}-#{System.unique_integer([:positive])}.json"
```